crates/runtime/src/dataconnector/github/pull_requests.rs

```diff
@@ -22,12 +22,54 @@ use std::sync::Arc;
 pub struct PullRequestTableArgs {
     pub owner: String,
     pub repo: String,
+    pub use_search: bool,
```

Comment:
Let's make this: `query_mode`

Default: `github_query_mode: auto`

Option: `github_query_mode: search`